SyncTests/RecordTests.swift

```diff
@@ -271,6 +277,60 @@ class RecordTests: XCTestCase {
         XCTAssertEqual("http://www.bbc.co.uk/go/rss/int/news/-/news/", m.siteURI)
     }
 
+    func testDeletedRecord() {
+        let json = JSON([
+            "id": "abcdefghijkl",
+            "deleted": true,
+            "type": "bookmark",
+        ])
+
+        guard let payload = BookmarkType.payloadFromJSON(json) else {
+            XCTFail()
+            return
+        }
+
+        XCTAssertFalse(payload is BookmarkPayload)    // Only BookmarkBasePayload.
+        XCTAssertTrue(payload.isValid())
+    }
+
+    func testUnknownRecordType() {
+        // It'll return a base payload that's invalid because its type is unknown.
```

Comment:
Counter-nit: you're wrong! It's not a possessive or contraction. "The sheep ate because its stomach was empty, and it's an animal."